docker-compose/README.md

```diff
@@ -58,12 +58,12 @@ Just continue with the _Build & Run (from within the ./docker-compose directory)
 - The _.env_ file should contain the following and the `PLATFORM_CONFIG_DIR` and the `MOSQUITTO_CONFIG_DIR` should be the absolute paths to _\<YOUR CONFIG PATH\>/platform_ and _\<YOUR CONFIG PATH\>/mosquitto_
 
   ```code
-  DOCKER_HTTP_PROXY=http://host.docker.internal:3128   (Proxy configuration)
```

Comment:
I propose to add a hint for this parameters (e.g. together with the description above)

Reply:
Please look in lines 75+76, there's the reference to the proxy documentation

Reply:
Yes it was only the "look and feel". The lines above describe what have to be defined and then the example is shown which has much more information in it then these two. Therefore it feels a little bit odd and decoupled but the information is available.

I think we could improve the documentation a lot more. So we could keep it out of scope for now